### ISSUE TITLE
Framing module: add more default crop ratios

### ISSUE THREAD

**Issue by pedrorrodriguez** [aurelienpierreeng/ansel#127]
There are a couple of aspect ratio options within the module all expressed x : y. When you need a custom aspect ratio, you have to use a slider with values 1-3. This is not particularly helpful when the target medium is a screen. It is annoying when you want to print. Sure, you can divide the long side by the short one and get the value you need, but one should not need a calculator to use the module.

I think we need a way to specify the desired values for long and short sides in a sane way.

**Comment by aurelienpierre** [comment]
You can perform the division directly by typing `w : h` (where w is the number of width and h the number of h) in the combobox.

**Comment by pedrorrodriguez** [comment]
Do you mean right after selecting "custom" from the drop-down?

**Comment by aurelienpierre** [comment]
Well, the intended use case is :

1. need a "standard" predefined crop ratio, for printing and framing: get it from the list
2. need to crop that fits your image content, for web and such: use "custom" crop
3. need a non-standard predefined crop ratio: WTF are you doing ? Oh well, type it, we'll compute it for you.

I think it makes sense.

**Comment by pedrorrodriguez** [comment]
Makes sense. The problem is that for 1. we don't have enough presets for common frame sizes. See my previous comment for reference, and how not all of those are in the drop-down. 

Common usecase: I buy a frame in ikea and I want to print a picture with that aspect ratio.

**Comment by Scorpion078** [comment]
you can always create a preset on your own and apply it

**Comment by pedrorrodriguez** [comment]
Yes, but by the time I make a preset, the module has already failed to meet use case 1. Usecase 2 is no issue. Usecase 3 is virtually irrelevant, if there are enough standard aspect ratios.

**Comment by aurelienpierre** [comment]
Ok so it's about getting more preset ratios

**Comment by pedrorrodriguez** [comment]
I still find the way of setting the custom aspect ratio a bit cumbersome, but more default crop ratios would make it a non issue with probably much less effort. 

TLDR. Yes

**Comment by pedrorrodriguez** [comment]
> You can perform the division directly by typing `w : h` (where w is the number of width and h the number of h) in the combobox.

No matter what I try I can't get this to work. Right click on the drop-down, right click on custom, right click on the aspect ratio slider.. Also, the tool tip says (w:h) instead of 'w:h'. Parenthesis and quotes are not supposed to be included, are they?

**Comment by Scorpion078** [comment]
Open the crop module, just left click on the field where it says Aspect. 
When you see the drop down with all the aspect ratios, that is where you just type your desired ratio. 
For  example 6:4 no parenthesis or space and hit enter

**Comment by aurelienpierre** [comment]
Ok, the misunderstanding is resolved : this is about the framing module, that adds white borders, not about the cropping module. Framing has indeed only a few options.

**Comment by aurelienpierre** [comment]
Needs a parameter change that will void backward compatibility, so, not for now.

**Comment by pedrorrodriguez** [comment]
Not a bug, moved to discussions
